code/cobre.py

```diff
@@ -15,8 +15,30 @@
 import argparse
 from pathlib import Path
 
-
-def process_data(csv_file_p, output_json_p):
+# Define metadata
+metadata = {
+    "participant_id": {
+        "original_field_name": "[none given]",
+        "description": "Unique identifier for each participant",
+    },
+    "age": {
+        "original_field_name": "Current Age",
+        "description": "Age of the participant in years",
+    },
+    "sex": {"original_field_name": "Gender", "description": "Sex of the participant"},
```

Comment:
for categorical variable, we normally provide the label and the explanation. In the current case, the label itself is self explanatory but still a list would be helpful just so people know what exists. 

```suggestion
    "sex": {
        "original_field_name": "Gender", 
        "description": "Sex of the participant",
        "levels": {
            "male": "male",
            "female": "female"
        }
    },
```
Search `participants.json example:` to see the actual example in bids
https://bids-specification.readthedocs.io/en/stable/modality-agnostic-files.html#participants-file